### DISCUSSION TITLE
Game cheat

### DISCUSSION THREAD

**Discussion by peyre** [raceintospace/raceintospace#597] [Ideas]
I'm not usually a fan of cheats, but I wonder if it would enrich the game to add one (say, "$" in R&D and Purchasing) to add, say, 100MB to your Cash.

**Comment by peyre** [comment]
Ha, I added it and it works (haven't uploaded of course).  "$" adds 100MB to your Cash on hand, but only in Purchasing.  Then it occurred to me that this could be a real liability in a two-player game!  How do I check the type of game so this only works in single-player?

**Reply by peyre** [reply]
Ah, got it.  I'll commit - if people think this wasn't such a good idea, feel free to revert it.

**Reply by hweimer** [reply]
Personally, I find this cheat very useful for debugging purposes. But how about making this optional, like the nofail cheat?

**Reply by peyre** [reply]
Sure, that sounds ideal.  Maybe it could be enabled by default?  I'm not sure how to go about adding preference items though, but I'll see if I can work it out.

**Reply by peyre** [reply]
Ok, it's now an option in the config file, and having the cheat disabled is a Classic setting.
